src/test/resources/test-rom/spec/modules/cc/expect_spec.lua

```diff
@@ -50,4 +50,22 @@ describe("cc.expect", function()
                 :eq("bad field 'l' (expected string, table or number, got boolean)")
         end)
     end)
+        
+    describe("range", function()
+        it("checks a single type", function()
+            expect(e.range(1, 1, 1)):eq(1)
+            expect(e.range(2, 1, 3):eq(2)
```

Comment:
```suggestion
            expect(e.range(2, 1, 3)):eq(2)
```

Sorry, missed this one!